src/app/filelogger.cpp

```diff
@@ -134,19 +134,19 @@ void FileLogger::addLogMessage(const Log::Msg &msg)
     switch (msg.type)
     {
     case Log::INFO:
-        stream << "(I) ";
+        stream << u"(I) ";
         break;
     case Log::WARNING:
-        stream << "(W) ";
+        stream << u"(W) ";
         break;
     case Log::CRITICAL:
-        stream << "(C) ";
+        stream << u"(C) ";
         break;
     default:
-        stream << "(N) ";
+        stream << u"(N) ";
     }
 
-    stream << QDateTime::fromMSecsSinceEpoch(msg.timestamp).toString(Qt::ISODate) << " - " << msg.message << '\n';
+    stream << QDateTime::fromMSecsSinceEpoch(msg.timestamp).toString(Qt::ISODate) << u" - " << msg.message << u'\n';
```

Comment:
Seems QTextStream consider char16_t as numeric type so there are problems with new lines for now.